mods/ra/maps/allies-09a/map.yaml

```diff
@@ -0,0 +1,863 @@
+	Actor224: ss
+		Owner: USSR
+		Location: 60,13
+		Facing: 163
+	Actor225: ss
+		Owner: USSR
+		Location: 61,12
+		Facing: 167
+	Actor226: ss
+		Owner: USSR
+		Location: 60,11
+		Facing: 158
+	Actor227: harv
+		Owner: USSR
+		Location: 39,3
+		Facing: 0
+	Actor228: 3tnk
+		Owner: USSR
+		Location: 27,3
+		TurretFacing: 193
+		Facing: 193
+	Actor229: 3tnk
+		Owner: USSR
+		Location: 28,4
+		TurretFacing: 193
+		Facing: 193
+	Actor230: dog
+		Owner: USSR
+		SubCell: 3
+		Location: 26,3
+		Facing: 179
+	Actor231: dog
+		Owner: USSR
+		SubCell: 3
+		Location: 26,5
+		Facing: 201
+	Actor232: e1
+		Owner: USSR
+		SubCell: 3
+		Location: 25,4
+		TurretFacing: 189
+		Facing: 189
+	Actor233: v2rl
+		Owner: USSR
+		Location: 19,6
+		Facing: 166
+	Actor234: v2rl
+		Owner: USSR
+		Location: 17,8
+		Facing: 166
+	Actor235: 3tnk
+		Owner: USSR
+		Location: 18,7
+		TurretFacing: 169
+		Facing: 169
+	Actor236: v2rl
+		Owner: USSR
+		Location: 9,5
+		Facing: 126
+	Actor237: v2rl
+		Owner: USSR
+		Location: 11,7
+		Facing: 130
+	Actor238: 3tnk
+		Owner: USSR
+		Location: 10,6
+		Facing: 128
+		TurretFacing: 128
+	Actor239: 4tnk
+		Owner: USSR
+		Location: 13,13
+		TurretFacing: 132
+		Facing: 132
+	Actor240: 4tnk
+		Owner: USSR
+		Location: 12,15
+		TurretFacing: 127
+		Facing: 127
+	Actor241: 4tnk
+		Owner: USSR
+		Location: 14,15
+		TurretFacing: 132
+		Facing: 132
+	Actor242: 3tnk
+		Owner: USSR
+		Location: 5,12
+		TurretFacing: 151
+		Facing: 151
+	Actor243: 3tnk
+		Owner: USSR
+		Location: 6,16
+		TurretFacing: 159
+		Facing: 159
+	Actor244: dog
+		Owner: USSR
+		Facing: 96
+		SubCell: 3
+		Location: 12,22
+	Actor245: dog
+		Owner: USSR
+		SubCell: 3
+		Location: 14,22
+		Facing: 154
+	Actor246: e1
+		Owner: USSR
+		Facing: 96
+		TurretFacing: 96
+		SubCell: 3
+		Location: 13,21
+	Actor247: 3tnk
+		Owner: USSR
+		Facing: 96
+		TurretFacing: 96
+		Location: 18,16
+	Actor248: 3tnk
+		Owner: USSR
+		Facing: 96
+		TurretFacing: 96
+		Location: 15,2
+	Actor249: e1
+		Owner: Greece
+		SubCell: 3
+		Location: 17,60
+		TurretFacing: 222
+		Facing: 222
+	Actor250: e1
+		Owner: Greece
+		SubCell: 3
+		Location: 23,62
+		TurretFacing: 29
+		Facing: 29
+	AlliedMCVEntry: waypoint
+		Owner: Neutral
+		Location: 19,70
+	Unload1: waypoint
+		Owner: Neutral
+		Location: 19,63
+	DefaultCameraPosition: waypoint
+		Owner: Neutral
+		Location: 21,60
+	HeliLandZone: waypoint
+		Owner: Neutral
+		Location: 21,62
+	KosyginSpawnPoint: waypoint
+		Owner: Neutral
+		Location: 14,6
+	KosyginExtractPoint: waypoint
+		Owner: Neutral
+		Location: 25,57
+	HeliWP01: waypoint
+		Owner: Neutral
+		Location: 58,70
+	HeliWP02: waypoint
+		Owner: Neutral
+		Location: 58,66
+	HeliWP03: waypoint
+		Owner: Neutral
+		Location: 16,55
+	LowerBaseWP: waypoint
+		Owner: Neutral
+		Location: 13,25
+	UpperBaseWP: waypoint
+		Owner: Neutral
+		Location: 33,4
+	USSRUnload1: waypoint
+		Owner: Neutral
+		Location: 53,53
+	USSRRFEntry: waypoint
+		Owner: Neutral
+		Location: 80,1
+	revealLowerBase: waypoint
```

Comment:
Style nit: Actor names should start with a capital letter.

Reply:
Fixed at [c9bbea5](https://github.com/OpenRA/OpenRA/commit/c9bbea5b361b63990c5c073f6ac271692cc2ea62)